### PR TITLE
Extended sender: simulated data

### PR DESCRIPTION
**Pull request by miguelgarcia**
Hey, this standard is a great initiative. We are exploring it at InOrbit since it seems like a really nice way to get data from different robots into our platform. As part of this exploration we extended the sender to be able to generate more complete status reports with changing data.

Besides this, the PR includes a couple of fixes/improvements that we believe are useful:

* Fixed missing import of `datetime` `timezone`
* Read URI from argv

Thanks !

### CODE REVIEW

**Comment by miguelgarcia** [comment]
Hi @cacioppoc @zac-dydek is creating a PR the best way to contribute to this repo or is there another preferred method ?

Thanks a lot, have a great week!

File: MassRobotics-AMR-Sender/client.py [hunk 1]
```diff
@@ -6,10 +6,14 @@
...
 import hashlib
-from datetime import datetime
+import random
+import math
+import sys
+from datetime import datetime, timezone, timedelta
 
-async def sendMessage():
-  uri = "ws://localhost:3000"
+OPERATIONAL_STATES = ["navigating", "idle", "disabled", "offline", "charging", "waiting", "loadingUnloading", "manualOveride"]
```
Comment (zac-dydek) [inline MassRobotics-AMR-Sender/client.py:14]: The operational states in our working document are similar, but not identical.  We split "waiting" into:

1. Waiting for human event (which probably covers your "loadingUnloading" state?)
2. Waiting for external event
3. Waiting for internal event

**Review by zac-dydek — CHANGES_REQUESTED** [review]
These are great additions, just one small change to make the operational state choices match the standard document.

File: MassRobotics-AMR-Sender/client.py (hunk 1 above)
Comment (miguelgarcia) [reply MassRobotics-AMR-Sender/client.py:14]: Hi @zac-dydek I copied these values from `/AMR_Interop_Standard.json`, which seems to don't exactly follow what's in the doc. I can piggyback that change into this PR. What do you think would be good names ?

1. `waitingHumanEvent`
2. `waitingExternalEvent`
3. `waitingInternalEvent`
??

Thanks!

Comment (zac-dydek) [reply MassRobotics-AMR-Sender/client.py:14]: I see, yes, it seems the .json file is slightly out-of-date.  Yes, let's make that change while we're at it.  The names you picked are fine with me!

Comment (miguelgarcia) [reply MassRobotics-AMR-Sender/client.py]: Thanks @zac-dydek I made the changes, could you review ?

Thanks!

**Review by zac-dydek — APPROVED** [review]
Looks great, thanks!

**Pull request merged**